File: app/src/main/java/com/zhaoweihao/mrtranslator/utils/Utility.java
package com.zhaoweihao.mrtranslator.utils;

import com.google.gson.Gson;
import com.zhaoweihao.mrtranslator.gson.Translate;

/**
 * Created by Zhaoweihao on 2017/4/13.
 */

public class Utility {
    public static Translate handleTranslateResponse(String response){
        Gson gson=new Gson();
        Translate translate=gson.fromJson(response,Translate.class);
        return translate;
    }

}

File: app/src/main/java/com/zhaoweihao/mrtranslator/utils/Utility.kt
package com.zhaoweihao.mrtranslator.utils

import com.google.gson.Gson
import com.zhaoweihao.mrtranslator.gson.Translate

/**
 * Created by Zhaoweihao on 2017/4/13.
 */
object Utility {
    fun handleTranslateResponse(response: String?): Translate {
        val gson = Gson()
        return gson.fromJson(response, Translate::class.java)
    }
}